templates/criteriaAndAssessment.js

```diff
@@ -0,0 +1,114 @@
+
+const baseAssessment = {
```

Comment:
@swetha-aggidevara we should move this inside module of criteria and do add comments on each of the object types.